src/pages/Login/Login.scss

```
@@ -0,0 +1,57 @@
+@mixin flex($display: flex, $align-items: center, $justify-content: center) {
+  display: $display;
+  align-items: $align-items;
+  justify-content: $justify-content;
+}
```

flex는 display가 꼭 flex여야 하니 첫번째 parameter는 없어도 될 것 같아요!

```suggestion
@mixin flex($align-items: center, $justify-content: center) {
  display: flex;
  align-items: $align-items;
  justify-content: $justify-content;
}
```

만약 column flex를 다루고 싶다면 아래 코드도 추가하면 좋을 것 같군요!
```scss
@mixin columnFlex($align-items: center, $justify-content: center) {
  display: flex;
  flex-direction: column;
  align-items: $align-items;
  justify-content: $justify-content;
}
```